### ISSUE TITLE
Available to work with AutoKeras?

### ISSUE THREAD

**Issue by akudo7** [plaidml/plaidml#202]
Hi thank you very very much for publishing this GREAT tool. 

Is the plaidML available to work with AutoKeras? something like ...

```
import plaidml.keras
plaidml.keras.install_backend()

from keras.datasets import mnist
import autokeras as ak

if __name__ == '__main__':
    (x_train, y_train), (x_test, y_test) = mnist.load_data()
    x_train = x_train.reshape(x_train.shape + (1,))
    x_test = x_test.reshape(x_test.shape + (1,))

    clf = ak.ImageClassifier(verbose=True, augment=False)
    clf.fit(x_train, y_train, time_limit=12 * 60 * 60)
    clf.final_fit(x_train, y_train, x_test, y_test, retrain=True)
    y = clf.evaluate(x_test, y_test)
    print(y * 100)
```

I am enjoying to play with plaidML on the latest macbook pro and metal_amd_radeon_pro_560x.0 : AMD Radeon Pro 560X (Metal).
Any information appreciate.

**Comment by brianretford** [comment]
Thanks! I'm not really sure about autokeras support. What happens when you try and use it?

**Comment by akudo7** [comment]
The GPU that plaidML uses seems to be not used.

**Comment by hemangjoshi37a** [comment]
Only the NVIDIA GPUs are supposed with CUDA cores.

**Comment by brianretford** [comment]
Could you try on the latest release and let us know if this still reproduces?

**Comment by dgkutnic** [comment]
Issues in the `needs info` state for 7+ days will be marked as obsolete and closed. If the issue is still occurring for you, please re-open the issue and provide us with the requested info.

**Comment by brzegorz** [comment]
Hi, reopening because I'm also interested. Running the following snippet:

`import autokeras as ak
import os
import tensorflow as tf
from keras import backend 
print(backend.backend())
print(tf.keras.backend.backend())`

gives output:

`plaidml
tensorflow`

tf.keras is the one being referenced in the codebase. Judging by times on MNIST, it seems that it's impossible to use autokeras and plaidml together - unless I'm doing something wrong. Full script:

`from sklearn import datasets
from sklearn.metrics import accuracy_score
from sklearn.ensemble import RandomForestClassifier
from sklearn.model_selection import train_test_split
import plaidml.keras
plaidml.keras.install_backend()
import autokeras as ak
import os

def load_mnist_data():
    # The digits dataset
    digits = datasets.load_digits()
    
    # To apply a classifier on this data, we need to flatten the image, to
    # turn the data in a (samples, feature) matrix:
    n_samples = len(digits.images)
    data = digits.images.reshape((n_samples, -1))
    return data, digits.target

X, y = load_mnist_data()

X_train, X_test, y_train, y_test = train_test_split(
            X, y, test_size=0.3, shuffle=False)
            
clf = ak.StructuredDataClassifier(max_trials=10) # It tries 10 different models.
clf.fit(X_train, y_train)
preds = clf.predict(X_test, y_test)
acc = accuracy_score(y_test, preds)
print("Accuracy of autokeras: " + str(acc))`
